pkg/operator/ceph/csi/version_test.go

```diff
@@ -23,11 +23,12 @@ import (
 )
 
 var (
-	testMinVersion         = CephCSIVersion{3, 5, 0}
-	testReleaseV340        = CephCSIVersion{3, 4, 0}
+	testMinVersion         = CephCSIVersion{3, 7, 0}
 	testReleaseV350        = CephCSIVersion{3, 5, 0}
 	testReleaseV360        = CephCSIVersion{3, 6, 0}
 	testReleaseV361        = CephCSIVersion{3, 6, 1}
+	testReleaseV370        = CephCSIVersion{3, 7, 0}
+	testReleaseV371        = CephCSIVersion{3, 7, 1}
```

Comment:
3.7.1 is not available ?

Reply:
This only for test to check do we allow update to minor version

Reply:
k. sounds fine